addons/mrp/views/mrp_workorder_views.xml

```diff
@@ -513,11 +513,9 @@
         <field name="context">{'search_default_ready': True, 'search_default_progress': True, 'search_default_pending': True}</field>
         <field name="help" type="html">
           <p class="o_view_nocontent_smiling_face">
-            Start a new work order
+            Plan a Manufacturing Order
           </p><p>
-            Work Orders are operations to be processed at a Work Center to realize a
-            Manufacturing Order. Work Orders are trigerred by Manufacturing Orders,
-            they are based on the Routing defined on these ones
+            Work orders correspond to the different operations, taking place at specific workcenters,which are performed to manufacture a product.Those can be defined in a routing that will be assigned to a bill of material.
```

Comment:
there's a missing space after a coma  and after a dot.
workcenters,which -> workcenters, which
manufacture a product.Those can  -> manufacture a product. Those can